documentation/src/main/asciidoc/cheat-sheets/cache-configuration.asciidoc

```diff
@@ -0,0 +1,435 @@
++
+[source,xml]
+----
+<distributed-cache mode="SYNC" />
+----
+Asynchronous replication:: Reduces latency and increases the speed of write operations but leads to data inconsistency and provides a lower guarantee against data loss.
++
+[source,xml]
+----
+<distributed-cache mode="ASYNC" />
+----
+
+== Cache configuration options
+
+.Encoding
+A format, identified by a media type, that {brandname} uses to store keys and values.
+
+TIP: {brandname} recommends the `application/x-protostream` media type. Protobuf gives you excellent performance as well as interoperability between client applications in different programming languages for both Hot Rod and REST endpoints.
+
+[source,xml]
+----
+<distributed-cache mode="SYNC">
+  <encoding>
+    <key media-type="application/x-protostream"/>
+    <value media-type="application/x-protostream"/>
+  </encoding>
+</distributed-cache>
+----
+
+.Expiration
+Controls how long {brandname} keeps entries in a cache and takes effect across clusters.
+
+Lifespan:: The maximum amount of time, in milliseconds, that cache entries can exist.
+If you set the value to -1, entries never expire.
+Lifespan uses a `creation` timestamp and the value for the `lifespan` property to determine when entries expire.
+The creation time for cache entries provides a value that is consistent across clustered caches.
+
+Max-idle:: The maximum amount of time, in milliseconds, that cache entries can remain idle.
+If no operations are performed on entries within the maximum idle time, the entries expire across the cluster.
+If you set the value to -1, entries never expire.
+
+Maximum idle uses a `last used` timestamp and the value for the `max-idle` property to determine when entries expire.
+The last access time for entries is not always the same on all nodes.
+To ensure that entries have the same relative access times across clusters, {brandname} sends touch commands to all owners when keys are accessed.
+This synchronous behavior increases latency of client requests.
+
+TIP: The {brandname} team recommends setting the value for `lifespan` to be at least double of the value that you set for the `max-idle` property. Configuring `max-idle` close to `lifespan` has performance drawbacks.
+
+[source,xml]
+----
+<distributed-cache>
+  <expiration lifespan="5000"
+              max-idle="1000" />
+</distributed-cache>
+----
+
+.Statistics
+Start collecting statistics for your cache.
+Keep disabled for optimal performance.
+
+[source,xml]
+----
+<distributed-cache statistics="true" />
+----
+
+== {brandname} capabilities
+* Bounded
+* Indexed
+* Authorization
+* Backups
+* Transactional
+* Persistent
+
+== Bounded caches
+Eviction controls the amount of data that {brandname} keeps in memory and takes effect for each node.
+Because not all nodes evict the same entries you should use eviction with persistent storage to avoid data inconsistency.
+
+.Eviction
+Control the size of the cache in one of the following ways:
```

Comment:
It does not control the size of the cache, it takes only effect on the local nodes memory and prevent from more than max-entries or more memory used than max-size.